plugins/analytics-module-ga4/README.md

```diff
@@ -61,7 +61,13 @@ backend:
   csp:
     connect-src: ["'self'", 'http:', 'https:']
     # Add these two lines below
-    script-src: ["'self'", "'unsafe-eval'", 'https://www.google-analytics.com']
+    script-src:
+      [
+        "'self'",
+        "'unsafe-eval'",
+        'https://www.google-analytics.com',
+        'https://www.googletagmanager.com',
+      ]
     img-src: ["'self'", 'data:', 'https://www.google-analytics.com']
```

Comment:
Do you know if it's needed here as well? Feeling it's a bit awkward for img-src, but GA is there already 😅

Reply:
Honestly, from what I've observed, neither is needed in `img-src` for this plugin to be functional. I tried running it with both of those removed and it seemed to work fine without any issues. However, I am far from being an expert with setting CSPs.

Here is the official Google Analytics documentation
https://developers.google.com/tag-platform/tag-manager/web/csp

Where they suggest:
```
script-src:  https://*.googletagmanager.com
img-src:     https://*.google-analytics.com https://*.googletagmanager.com
connect-src: https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com
```

But the bare minimum that seems like it is needed for this to work is just adding googletagmanager.com to `script-src`. The default `connect-src` we have in example `app-config.yaml` already globally allows all connections.